File: tests/scripts/subcommands/compile_run_crun/stage-without-grader/solution/correct-args1.java
import java.util.Scanner;

class add1 {
	public static void main(String... args) {
		int argc = args.length;
		System.out.println("#args="+argc);
		for (int i = 0; i < argc; i++)
			System.out.println("arg["+(i+1)+"]='"+args[i]+"'");
		Scanner sc = new Scanner(System.in);
		int a = sc.nextInt();
		int b = sc.nextInt();
		System.out.println(a+b);
	}
}

File: tests/scripts/subcommands/compile_run_crun/stage-without-grader/solution/correct-args1.cpp
#include <iostream>
using namespace std;

int main(int argc, char** argv) {
	printf("#args=%d\n", argc-1);
	for (int i = 1; i < argc; i++)
		printf("arg[%d]='%s'\n", i, argv[i]);
	int a, b;
	cin >> a >> b;
	cout << a+b << endl;
	return 0;
}
